src/views/Backups/BackupsView.tsx

```diff
@@ -80,46 +83,83 @@ export default function BackupsView(): JSX.Element {
     }
   };
 
-  const onDeleteSelectedItems = async () => {
-    const selectedItemsIDs = new Set(selectedItems.map((item) => item.id));
-    const filteredCurrentItems = currentItems.filter((item) => !selectedItemsIDs.has(item.id));
+  const openDeleteItemsDialog = (items: DriveItemData[]) => {
+    if (!items.length || isFileViewerOpen) {
```

Comment:
why if the file viewer is opened we skip the dialog and we delete the items directly? 
I think that we should it display always